rhobs/alerting/data_plane/prometheus.gitops_cluster_agent_op_alerts.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-cluster-agent-operations-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: cluster-agent-operations_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsClusterAgentOperationErrors
+      expr: |
+        sum(operationDB_rows_in_non_complete_state) <= 10
```

Comment:
Question regarding this metric and also in the grafana dashboard visualization.  Is this metric already the sum total of the number of rows in non-complete state?  Comparing this to, for example, the metric `operationDB_rows_in_completed_state` which returns a positive integer number >=0.  In the sandbox member cluster, this number currently returns 288.

Reply:
yes @keithchong The metric `operationDB_rows_in_non_complete_state`  returns the total number of operation db rows in non-complete state, if we take an example of operationDB_rows_in_completed_state , in  prod multitenant grafana dashboard, its value is 144, In the multitenant grafana  dashboard we are checking the values from last 1 hour, so it will return the total operationDB_rows_in_completed_state  from last 1 hour i.e 144, similar case is with operationDB_rows_in_non_complete_state it will return total number of  operation db rows in non-complete state from last 1 hour, as time-range is set for 1 hour, but in RHTAP SLO dashboard the time range is set for 28d so it return multiple values i.e, it will return all the values from past 28d like this 
<img width="1274" alt="Screenshot 2023-11-20 at 12 52 29 PM" src="https://github.com/redhat-appstudio/o11y/assets/76523858/9c0a96a5-9236-4476-a9f2-f992bc14e81c">

So I wanted to sum up all the `operationDB_rows_in_non_complete_state` values from past 28d

Reply:
Is there any other way to aggregate the data from past 28d based on time range or we can add the time limit by ourselves like this, `sum(increase(operationDB_rows_in_non_complete_state[24h]))`, please let me know any other possible ways to do this

Reply:
Is state complete means active or state non-complete means active?

I'm not sure what are we trying to check. What's the reason for aggregating over time?
If I understand correctly, this metric is a gauge. What's the meaning of summing its value over time?

Reply:
If it should be always less than 10, do we need to sum over time? The requirement seem to be about the current value. I don't see it implemented by summing over time.

Reply:
And in ticket it is mentioned as <=10 cluster-agent operations should b active `at any given time`

Reply:
The alert is being evaluated every minute. At the time of evaluation, the alert query is run and either returns some time series or not.
Depending on the value given for the `for` field, a time series need to be returned in multiple consecutive such evaluations before the alert will fire. The `for` field is there to eliminate noise.

With that in mind, I think this query needs to be without the sum, and the `for` value should be set according to what you see fit.